sampler.go

```diff
@@ -293,6 +324,7 @@ func NewAdaptiveSampler(strategies *sampling.PerOperationSamplingStrategies, max
 		sampler, err := NewGuaranteedThroughputProbabilisticSampler(
 			strategies.DefaultLowerBoundTracesPerSecond,
 			strategy.ProbabilisticSampling.SamplingRate,
+			defaultUpperBound, // TODO (wjang) get the upperbound from strategy
```

Comment:
function comment: "NewAdaptiveSampler adaptiveSampler is a delegating "

should be "NewAdaptiveSampler {verb} ..."